src/main/java/com/xiaomi/infra/pegasus/client/PegasusScanner.java

```diff
@@ -204,10 +205,15 @@ private void onRecvRpcResponse(error_code err, scan_response response) {
       if (response.error == 0) { // ERR_OK
         _kvs = response.kvs;
         _p = -1;
-        _context = response.context_id;
+        _contextId = response.context_id;
       } else if (response.error
           == 1) { // rocksDB error kNotFound, that scan context has been removed
-        _context = CONTEXT_ID_NOT_EXIST;
+        _contextId = CONTEXT_ID_NOT_EXIST;
+      } else if (response.error == 7) { // rocksDB error kIncomplete
```

Comment:
What will happen if the server with scan-limit meets with the client of the previous version? Will the client throw an exception when it accepts `rocksdb::Status::kIncomplete` or just continue?

Reply:
In current version, client will throw an exception when it accepts `rocksdb::Status::kIncomplete` immediately and can not get the result of the scanner. In this version, client will also throw an exception but will return scan result.